### DISCUSSION TITLE
"Unavailable"

### DISCUSSION THREAD

**Discussion by gohan2091** [RupertAvery/DiffusionToolkit#233] [Q&A] [answered]
In my imported folders, my images are showing but I have a lot of tiles that say "unavailable". How do I clear these? I would suspect that these are deleted images and I just need Diffusion Toolkit to update the files it sees.
![Screenshot 2024-03-31 173818](https://github.com/RupertAvery/DiffusionToolkit/assets/127342824/7e5d4f24-4f3a-4ebc-891b-ac98d7781dcc)

**Comment by rtaskf** [comment, marked as answer]
I'm not sure the developer is visiting this forum. The only way I found to get rid of the "unavailable" thumbnails is to delete the database file "diffusion-toolkit.db" under AppData->Roaming->DiffusionToolkit. It will then be rebuilt on the next start and without the unavailable files.

**Reply by gohan2091** [reply]
That did it, thanks!

**Comment by RupertAvery** [comment]
The latest commits has **Tool > Unavailable Files** for dealing with this for now. 

This build artifact should  have these changes, if you don't want to wait for an official build:

https://github.com/RupertAvery/DiffusionToolkit/actions/runs/8773444541/artifacts/1433316835

Latest builds will appear in https://github.com/RupertAvery/DiffusionToolkit/actions. Click on the workflow run and scroll down to the artifacts and download DiffusionToolkit.

I'm a bit more busy with work, so I haven't really had time to work on Diffusion Toolkit.

**Reply by gohan2091** [reply]
Thanks! just installed it and it works great. I was using an application called Breadboard before I found Diffusion Toolkit. Are you familar with Breadboard? It seems like an abandoned project but I find Diffusion Toolkit better although Breadboard has some interesting features if you ever want to check it out. You want want to incorporate some of their features here.
